src/engine/console.cpp

```diff
@@ -506,7 +507,21 @@ bool consolekey(int code, bool isdown)
 
             case SDLK_BACKSPACE:
             {
-                int len = (int)strlen(commandbuf), i = commandpos>=0 ? commandpos : len;
+                size_t commandbuf_len = strlen(commandbuf);
+                if (holding_alt)
+                {
+                    char* last_word = strrchr(commandbuf, ' ');
+                    if (last_word)
+                    {
+                        commandbuf[(commandbuf_len - strlen(last_word))] = 0;
+                    }
+                    else
+                    {
+                        commandbuf[0] = 0;
+                    }
+                }
+
+                int len = (int)commandbuf_len, i = commandpos>=0 ? commandpos : len;
```

Comment:
Consider adding more blank lines to group what's logically related. I'd add one below here.

Reply:
done.

Reply:
Please feel free to just mark conversations as resolved when you're done.